src/endpoint/s3/ops/s3_get_object.js

```diff
@@ -53,15 +55,13 @@ async function get_object(req, res) {
         obj_id: object_md.obj_id,
         bucket: req.params.bucket,
         key: req.params.key,
-        version_id: req.query.versionId,
+        version_id: version_id,
```

Comment:
short property notation FTW -
```suggestion
        version_id,
```